<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dark Text</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelImage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Выберите изображение&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="font">
          <font>
           <family>Source Code Pro</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="CancelButton">
        <property name="styleSheet">
         <string notr="true">   QPushButton {
        border: 1px solid #CCCCCC;
       background-color:#d5d5d5;
        border-color: #55aaff;
    	border-radius: 4px;
        color: #000000;
        font-size: 14px;
    	text-align:center;
        padding: 4px 6px;
        vertical-align: middle;
    	text-decoration:none;
 margin: 5px;
    }

 QPushButton:hover {
       color: #000000;
       background-color:#7bacda;
       outline: 0 none;
    }

QPushButton:pressed {
      border-color:  #4b66ff;
       background-color:#7bacda;
      color: #000000;
}</string>
        </property>
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ReadButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">   QPushButton {
        border: 1px solid #CCCCCC;
       background-color:#d5d5d5;
        border-color: #55aaff;
    	border-radius: 4px;
        color: #000000;
        font-size: 14px;
    	text-align:center;
        padding: 4px 6px;
        vertical-align: middle;
    	text-decoration:none;
 margin: 5px;
    }

 QPushButton:hover {
       color: #000000;
       background-color:#7bacda;
       outline: 0 none;
    }

QPushButton:pressed {
      border-color:  #4b66ff;
       background-color:#7bacda;
      color: #000000;
}</string>
        </property>
        <property name="text">
         <string>Прочитать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddTextButton">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">   QPushButton {
        border: 1px solid #CCCCCC;
       background-color:#d5d5d5;
        border-color: #55aaff;
    	border-radius: 4px;
        color: #000000;
        font-size: 14px;
    	text-align:center;
        padding: 4px 6px;
        vertical-align: middle;
    	text-decoration:none;
 margin: 5px;
    }

 QPushButton:hover {
       color: #000000;
       background-color:#7bacda;
       outline: 0 none;
    }

QPushButton:pressed {
      border-color:  #4b66ff;
       background-color:#7bacda;
      color: #000000;
}</string>
        </property>
        <property name="text">
         <string>Добавить текст</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">QStatusBar {
  background: #FFFFFF;
 color:  #000000;
}

QStatusBar QToolTip {
  background-color: #FFFFFF;
  color: #000000;
  padding: 0px;
  opacity: 230;
}

QStatusBar QLabel {
  background: transparent;
}</string>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="open"/>
    <addaction name="save_as"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Документ</string>
    </property>
    <addaction name="open_doc"/>
    <addaction name="save_a_doc"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="save_as">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="open_doc">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="save_a_doc">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
